plugins/controlboard/include/yarp/dev/ControlBoardDriver.h

```diff
@@ -403,6 +403,7 @@ class yarp::dev::GazeboYarpControlBoardDriver:
 
     yarp::sig::Vector m_positions; /**< joint positions [Degrees] */
     yarp::sig::Vector m_velocities; /**< joint velocities [Degrees/Seconds] */
+    yarp::sig::Vector m_accelerations; /**< joint accelerations [Degrees/Seconds/Seconds]> */
```

Comment:
I don't know the precedence rules between the `/` operator, but I guess that `Degrees/Seconds^2` would be less ambigous!